### PR TITLE
fix(AOT): Unmark Input fields as private

### PR DESCRIPTION
**Pull request by MikeRyan52**

### CODE REVIEW

File: src/lib/button/button.ts [hunk 1]
```diff
@@ -34,7 +33,7 @@ import {MdRippleModule} from '@angular2-material/core';
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class MdButton {
-  private _color: string;
+  @Input('color') _color: string;
```
Comment (jelbourn) [inline src/lib/button/button.ts]: This `@Input` should go on the getter for `color`

**Comment by jelbourn** [comment]
Thanks for the fix! Just one minor comment

**Comment by MikeRyan52** [comment]
@jelbourn Same for `sidenav.ts` ?

**Comment by jelbourn** [comment]
@MikeRyan52 yes, though it looks like it was already in the wrong place, so that would prbably fix another bug as well.

**Comment by MikeRyan52** [comment]
@jelbourn 👍 Fixed the location of both input decorators.

**Comment by jelbourn** [comment]
LGTM

**Comment by jelbourn** [comment]
Looks like that sidenav change actually started making some unit tests fail. Go ahead and move the `@Input` back to the `_opened` property and we'll resolve it later.

**Pull request closed without merging**